gno/p/role_manager/role_manager.gno

```diff
@@ -0,0 +1,187 @@
+package role_manager
+
+import (
+	"std"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/ownable"
+)
+
+const (
+	RoleUser  = "user"
+	RoleAdmin = "admin"
+)
+
+const (
+	PermissionAll       = "all"
+	PermissionWriteRole = "write:role"
```

Comment:
you did not address morgan's comments I think https://github.com/TERITORI/teritori-dapp/pull/1285#discussion_r1787527009 and https://github.com/TERITORI/teritori-dapp/pull/1285#discussion_r1787527738

Reply:
resolve partially in: [aa249ec](https://github.com/TERITORI/teritori-dapp/pull/1293/commits/aa249ec4a581a01be7407b0490bca3f87c25c564)